src/components/dialog/dialog.js

```diff
@@ -1142,7 +1142,9 @@ function MdDialogProvider($$interimElementProvider) {
           for (var i = 0; i < children.length; i++) {
             // skip over child if it is an ascendant of the dialog
             // or a script or style tag
-            if (element !== children[i] && !isNodeOneOf(children[i], ['SCRIPT', 'STYLE'])) {
+            if (element !== children[i] &&
+             !isNodeOneOf(children[i], ['SCRIPT', 'STYLE']) &&
+             !children[i].hasAttribute('aria-live')) {
```

Comment:
Can you also add a unit test for this change?

Here's an existing test for this behavior: https://github.com/angular/material/blob/e96293a7010e63e381336f4e0928057eb7880bd1/src/components/dialog/dialog.spec.js#L1690

You should be able to follow that to create a new test